src/components/Form.js

```diff
@@ -1,34 +1,67 @@
-import { useState } from 'react'
+import { useState, useEffect, createRef } from 'react'
 import styled from 'styled-components/macro'
 import PropTypes from 'prop-types'
 import Button from './Button'
 import Input from './Input'
+import { CrossIcon } from '../lib/Icons'
 
 function Form({ userPlace, setUserPlace, status }) {
     const [userInput, setUserInput] = useState('')
+    const [validation, setValidation] = useState(true)
+
+    const textInput = createRef()
+
+    useEffect(() => setValidation(true), [userInput])
 
     function handleChange(event) {
         setUserInput(event.target.value)
     }
 
     function handleSubmitPlace(event) {
         event.preventDefault()
-        userInput === userPlace
-            ? setUserPlace(userInput + ' ')
-            : setUserPlace(userInput)
+        if (validateInput()) {
+            userInput === userPlace
+                ? setUserPlace(userInput + ' ')
+                : setUserPlace(userInput.trim())
```

Comment:
It's an uncommon way to use `?` `:` as a replacement for `if` `else`. See https://javascript.info/ifelse#non-traditional-use-of
Instead, I would change it to this:

```suggestion
            setUserPlace(userInput === userPlace ? userInput + ' '  : userInput.trim())
```

I don't understand why you add a whitespace here if `userInput` is equal to `userPlace`. Maybe this is something you can reconsider :).